pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/base/AbstractBaseConnection.java

```diff
@@ -51,7 +51,7 @@ public void abort(Executor executor)
   @Override
   public void clearWarnings()
       throws SQLException {
-    throw new SQLFeatureNotSupportedException();
+    // no-op
```

Comment:
This seems unrelated to this PR. Can you open a separate PR for these changes?

Reply:
unfortunately these are part of the changes for #11813 that i based this PR on.